crates/light_sdk/Cargo.toml

```diff
@@ -19,6 +19,7 @@ blocking = ["tokio"]
 namada-eth-bridge = ["namada_sdk/namada-eth-bridge"]
 
 [dependencies]
+namada_io = { path = "../io" }
```

Comment:
ah sry to be pain - there's one more we can skip I think

Reply:
it's fixed